### ISSUE TITLE
Add a compiled SmartPutty.jar to repository

### ISSUE THREAD

**Issue by crisistotal2014** [ericmore/SmartPutty#23]
Hi Eric:
What do you think about adding a compiled SmartPutty.jar to your repository folder?

This is due many people doesn't have Gradle or JDK installed, and this can be a fast way to these people to check your great program.

P.D.:
Perhaps an "SmartPutty.exe" will be faster and easy to use this program.

**Comment by ericmore** [comment]
Frankly speaking your idea is good, however several years ago , I built a release to github, it is a one click runner with bat + jar.
But recently I found that release tab get missing. May need to explorer another way to do release, however checkin binary into repository is not a good idea,  if you are aware, my git report is pretty huge(abnormal) due to at early stage I embedded jdk inside the repo, even now I deleted it,  the repo is still large.

**Comment by ericmore** [comment]
You can understand more by reading this Article : https://medium.com/@sangeethkumar.tvm.kpm/cleaning-up-a-git-repo-for-reducing-the-repository-size-d11fa496ba48

**Comment by ericmore** [comment]
Oh, found it,the old release there https://github.com/ericmore/SmartPutty/releases/tag/1.0.0

BTW, I've merge your branch into master, the cleaned those unecessary branches.

**Comment by crisistotal2014** [comment]
YES! Exactly this I've missed: quick links to download your program (today in 32 and 64 bits version, obviously).
I suppose GitHub continues allowing to individual upload files to repository to download in cases likes this, no?
